index.js

```diff
@@ -118,6 +143,6 @@ async.auto({
   }
 
   if (results.containers) {
-    console.log(chalk.green('No containers found. Closing.'));
+    console.log(chalk.green('All containers have been modified since ' + chalk.bold(ageWindowStartString) + '. Closing.'));
```

Comment:
This part is still wrong in my fork...it prints at the end of a run with mixed removals. :/